deserializer.go

```diff
@@ -4,9 +4,12 @@ import (
 	"bytes"
 	"encoding/gob"
 	"regexp"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/getlantern/golog"
+	iradix "github.com/hashicorp/go-immutable-radix"
```

Comment:
Out of curiosity, why did you choose an immutable radix tree instead of the mutable version?